astro/src/content/quickstarts/quickstart-dotnet-web.mdx

```diff
@@ -0,0 +1,325 @@
+
+Copy the following code into `Account.cshtml.cs`:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Account.cshtml.cs"}
+  lang="c#" />
+
+Copy the following code into `Account.cshtml`:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Account.cshtml"}
+  lang="razor" />
+
+Create `MakeChange.cshtml` and `MakeChange.cshtml.cs` in the `Pages` directory. You will use the same [Authorize filter attribute](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/filter?view=aspnetcore-3.1#authorize-filter-attribute) to secure the page.
+
+Copy the following code into `MakeChange.cshtml.cs`:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/MakeChange.cshtml.cs"}
+  lang="c#" />
+
+Copy the following code into `MakeChange.cshtml`:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/MakeChange.cshtml"}
+  lang="razor" />
+
+You will need a way to allow the user to logout of the application as well. You'll build the logout functionality in the `Logout` page. This will:
+
+- Remove the session cookie.
+- Redirect to the FusionAuth OAuth logout endpoint.
+
+FusionAuth will then destroy its session and redirect the user back to the app's configured logout URL.
+
+Add the following file to the `Pages` directory and call it `Logout.cshtml.cs`:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Logout.cshtml.cs"}
+  lang="c#" />
+
+`OnGet` is the important method. Here you sign out using a method of the authentication library, delete the JWT cookie, and send the user to the FusionAuth OAuth logout endpoint.
+
+Now add `Logout.cshtml` to the `Pages` directory. No content is necessary. Just declare the page and model as shown below.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Logout.cshtml"}
+  lang="razor" />
+
+### Application Configuration
+You will need to do a little plumbing to ensure things like having the correct settings and configuring the cookies properly to work over http for the sample application.
+
+#### App Settings
+Update the `appsettings.json` file. You need to add the entire `complete-application` object so that the code above can be configured correctly. `Authority` is just the location of the user identity server, in this case, FusionAuth.  `Urls` ensures the application runs on the specified port, 5000.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/appsettings.json"}
+  lang="json" />
+
+#### Program Settings
+Update the `Program.cs` file. Since you are using unsecured cookies, you need to allow this in the configuration.  Safari and Chrome require different settings and that is reflected in the code below.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Program.cs"}
+  lang="c#" />
```

Comment:
Is it worth noting that for a production application you won't need to do this? (Or want to, because of security concerns.)

Reply:
Ah, I see you do this later. Good work!